lib/user.rb

```diff
@@ -0,0 +1,10 @@
+require 'pg'
+
+class User
+  def initialize(name, pw, email)
+    connection = PG.connect(dbname: 'Chitter')
+
+    connection.exec("INSERT INTO user_list (username, password, email) VALUES ('#{name}', '#{pw}', '#{email}');")
+  end
+
+end
```

Comment:
I like how you have made a user model and how it collaborates with your peep model